docs/docs/documentation/getting-started/installation/security.md

```diff
@@ -29,7 +29,6 @@ If you'd like to mitigate this risk, we suggest that you rate limit the API in g
 
 - [Traefik](https://doc.traefik.io/traefik/middlewares/http/ratelimit/)
 - [Nginx](https://nginx.org/en/docs/http/ngx_http_limit_req_module.html)
-- [Caddy](https://caddyserver.com/docs/modules/http.handlers.rate_limit)
```

Comment:
I don't think this should be removed, it's not a dangling reference, but a security suggestion

Reply:
Ah yeah you're right, thanks!